src/jit/hwintrinsicArm64.h

```diff
@@ -34,7 +34,8 @@ struct HWIntrinsicInfo
     // Reduce the number of forms
     enum Flags
     {
-        None
+        None          = 0,
```

Comment:
Prefix with `HW_Flag_`?

Reply:
It is inside `HWIntrinsicInfo` so it is `HWIntrinsicInfo::None`